src/sentry/static/sentry/app/views/ruleEditor/index.jsx

```diff
@@ -162,32 +176,30 @@ const RuleEditor = createReactClass({
             <h6>{t('Rule name')}:</h6>
             <div className="control-group">
               <input
-                ref="name"
                 type="text"
                 className="form-control"
                 defaultValue={name}
                 required={true}
                 placeholder={t('My Rule Name')}
+                onChange={e => this.updateRule('name', e.target.value)}
               />
             </div>
 
             <hr />
 
             <div className="node-match-selector">
-              <h6>
+              <h6 style={{display: 'flex', alignItems: 'center'}}>
```

Comment:
I would extract this into a styled component that's local to this module(since we re-use it elsewhere in this component) and maybe change to not use an `h6`